pkg/asset/templates/content/openshift/cluster-network-crds.go

```diff
@@ -0,0 +1,69 @@
+package openshift
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	netCRDfilename   = "cluster-network-crd.yaml"
```

Comment:
Either separate this asset into two separate assets or combine the files into a single file.

Reply:
`yamllint` blocks single files. Are you okay with me changing the `yamllint` configuration?